addons/website_slides/models/slide_channel.py

```diff
@@ -23,7 +23,7 @@ class ChannelUsersRelation(models.Model):
     partner_id = fields.Many2one('res.partner', index=True, required=True, ondelete='cascade')
     partner_email = fields.Char(related='partner_id.email', readonly=True)
 
-    def _recompute_completion(self):
+    def _recompute_completion(self, compute_karma_channel_finish=True):
```

Comment:
Would be good to write a docstring for this methods and add explanation about compute_karma_channel_finish.